src/ui/dialogs/settings.ts

```diff
@@ -37,15 +37,38 @@ class StartPage extends Page {
 class ClientSection extends Section {
    protected generateContentElement(): JQuery {
       let contentElement = new List();
+      contentElement.append(new ListItem(
+         Translation.t('After_changing_this_option_you_have_to_reload_the_page'),
+         undefined,
+         undefined,
+         undefined,
+         undefined
+      ));
 
       contentElement.append(new ListItem(
          Translation.t('Language'),
-         Translation.t('After_changing_this_option_you_have_to_reload_the_page'),
+         undefined,
```

Comment:
Why was this removed?

Reply:
just for design reasons but will revert it...